ext/Pod-Html/lib/Pod/Html/Util.pm

```diff
@@ -149,7 +126,8 @@ END_OF_USAGE
 
 =head2 C<unixify()>
 
-TK
+Provide the full path to a file, taking into account OS-specific
+peculiarities.
```

Comment:
```suggestion
When passed the full native path to a file, this routine returns it in a unix-like format.
```

Reply:
> 

Between the time of your initial review and now I committed other changes to the documentation of the helper subroutines in `lib/Pod/Html.pm`.  So you should probably work from those.